README.md

```diff
@@ -76,10 +77,37 @@ providing an API to convert user-entered number and date strings - in their
 own format - into actual numbers and dates, and conversely, to format numbers
 and dates into that string format.
 
+Even if the application deals only with the English locale, it may still need
+globalization to format programming language bytes into human-understandable
+language and vice-versa in an effective and reasonable way. For example, to
+display something better than "Edited 1 minutes ago".
+
+<a name="why_globalize"></a>
+### Why Globalize?
+
+Globalize provides number formatting and parsing, date and time formatting and
+parsing, currency formatting, message formatting (ICU message format pattern),
+and plural support.
+
+Our design goals are (a) to leverage the official CLDR JSON data, (b) allow
+users to load as much or as little data as they need, (c) avoid duplicating data
+if using multiple i18n libraries that leverage CLDR, (d) run in browsers and
+node.js and (e) make globalization as easy to use as jQuery.
+
+Some of the strengths of using Globalize are:
+
+- Based on CLDR data (largest and most extensive i18n repository).
+- No bugs due to outdated i18n content (it has no hard coded i18n data).
+- Consistency. Same results are obtained across browsers and on the server.
+- Support using multiple locales simultaneously.
+
```

Comment:
About making the list more parallel, what do you think of having all items as imperatives?

- Use the largest and most extensive i18n repository. Globalize is based on CLDR
  data.
- Avoid bugs caused by outdated i18n content. Globalize has no hard-coded i18n
  data. Up-to-date CLDR data can be used directly as soon as it's released.
- Get same results across different browsers. Get same results between browsers
  and the server.
- Use multiple locales simultaneously.